content/projects/morse-code/_index.md

```diff
@@ -59,7 +59,8 @@ Your directory structure should look like this:
 
 ## Instructions
 
-Create two functions:
+- Create an object of that contains morsecode. See resources below for examples. Make sure that you create only one object.
```

Comment:
The English isn't good.

Also this seems spoonfeedy. Can't we rather let the learner think for themselves?